src/index.js

```diff
@@ -20,7 +20,7 @@ function buildBundles (configs) {
   const promises = configs.map(buildBundle)
   return Promise.all(promises)
     .then((bundles) => {
-      console.log('built %d bundles', configs.length)
+      console.log('[%s] built %d bundles', new Date().toTimeString().substr(0, 8), configs.length)
```

Comment:
Does this mean this issue should re-open since we are focusing on it again @bahmutov ?

https://github.com/bahmutov/rollem/issues/10